### PR TITLE
Python updates for August release

### PR DESCRIPTION
**Pull request by kraigb**
To be merged when August release is made

### CODE REVIEW

File: docs/python/debugging.md [hunk 1]
```diff
@@ -171,7 +191,7 @@ An array of additional options that may contain the following:
 | `"DebugStdLib"` | Enabled debugging of standard library functions. |
 | `"Django"` | Activates debugging features specific to the Django web framework. |
 | `"Sudo"` | When used with `"console": "externalTerminal"`, allows for debugging apps that require elevation. Using an external console is necessary to capture the password. |
-| `"Pyramid"` | Used when debugging a Pyramid application. |
+| `"Pyramid"` | When set to true, ensures that a Pyramid app is launched with [the necessary pserve command](https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/startup.html?highlight=pserve). |
```
Comment (brettcannon) [inline docs/python/debugging.md]: I would make `pserve` be marked as code as it's a specific command.

File: docs/python/debugging.md [hunk 2]
```diff
@@ -223,11 +243,11 @@ Remote debugging allows you to step through a program locally within VS Code whi
 
 1. Local computer: switch to Debug View in VS Code, select the **Python: Attach** configuration, then select the settings (gear) icon to open `launch.json` to that configuration.
 
-1. Local computer: Modify the configuration so that `remoteRoot` provide the location of the program on the remote computer's file system. Also modify `host`, `port`, and `secret` to match the values in the `ptvsd.enable_attach` call added to the source code, except that you need to use the remote computer's public IP address for `host`. You might also change `name` to specifically identify the configuration. For example:
+1. Local computer: Modify the configuration so that `remoteRoot` provide the location of the program on the remote computer's file system. Also modify `host`, `port`, and `secret` to match the values in the `ptvsd.enable_attach` call added to the source code, except that you need to use the remote computer's public IP address for `host`. You might also change `name` to specifically identify the configuration that include the IP address. For example:
```
Comment (brettcannon) [inline docs/python/debugging.md]: "provide" -> "provides"

File: docs/python/debugging.md (hunk 1 above)
Comment (kraigb) [reply docs/python/debugging.md]: Will do.

File: docs/python/debugging.md (hunk 2 above)
Comment (kraigb) [reply docs/python/debugging.md]: Done, thanks.

**Pull request merged**